server/data/incidentReportingApi.ts

```diff
@@ -136,6 +137,14 @@ export type PaginationSortingParams = {
   sort: string[]
 }
 
+export type DescriptionAddendum = {
+  createdBy: string | null
```

Comment:
This is not nullable, the field is always populated (but for some records it may be the SYSTEM USERNAME)